aws/v9.3.4/release.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: v9.3.4
+spec:
+  state: active
+  date: 2020-07-07T15:00:00Z
+  apps:
+    - name: cert-exporter
+      version: 1.2.3
+    - name: chart-operator
+      version: 0.13.0
+    - name: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.16.0
+    - name: coredns
+      componentVersion: 1.6.5
+      version: 1.1.10
+    - name: kube-state-metrics
+      componentVersion: 1.9.5
+      version: 1.1.0
+    - name: metrics-server
+      componentVersion: 0.3.3
+      version: 1.1.0
+    - name: net-exporter
+      version: 1.9.0
+    - name: nginx-ingress-controller
+      componentVersion: 0.33.0
+      version: 1.7.0
+    - name: node-exporter
+      componentVersion: 0.18.1
+      version: 1.2.0
+  components:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 5.7.1
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      version: 0.23.10
+    - name: kubernetes
+      version: 1.16.9
+    - name: containerlinux
+      version: 2345.3.1
+    - name: calico
+      version: 3.10.1
```

Comment:
Did Firecracker make the required changes in aws-operator here? For including newer Calico https://github.com/giantswarm/giantswarm/issues/11307

Reply:
@giantswarm/team-firecracker is there aws-operator legacy release with calico upgrade?

Reply:
The specified aws-operator, v5.7.1, uses k8scloudconfig v6.1.1 which accepts Calico version from the Release CR so this should work. See https://github.com/giantswarm/k8scloudconfig/blob/v6.1.1/files/k8s-resource/calico-all.yaml#L380

Reply:
Thanks @tfussell, will give it another try. Last time I was testing it, it didn't work well with cluster upgrades - calico was remaining on same old version from previous platform release.

Reply:
Yes, it should work. Let me know if you have a cluster where it doesn't work, and I'll take a look.

Reply:
It worked this time. Included calico 3.10.4 upgrade in the 9.3.4 release and release notes.